test/functional/page_objects/common_page.ts

```diff
@@ -409,6 +410,21 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
       });
       return response.status !== 200;
     }
+
+    async isCloud() {
```

Comment:
Is the return type inferred correctly by Typescript for this? If not, maybe it needs to be explicit: `Promise<boolean>`

Reply:
I didn't see this used elsewhere in the file, but happy to add it

Reply:
Added in https://github.com/elastic/kibana/pull/51817/commits/01defaca130bb9be1f265f36c78b8f8933028230